### PR TITLE
remove sim1h and n3h

### PR DESCRIPTION
**Pull request by neonphog**
## PR summary

- Removed networking support for `sim1h`
- Removed networking support for `n3h`

There are still some awkwardly integrated references, this was the majority broad stroke.

## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

- [ ] if this is a code change that effects some consumer (e.g. zome developers) of holochain core,  then it has been added to [our between-release changelog](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md) with the format 

```markdown
- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)
```

## documentation

- [ ] this code has been documented according to our [docs checklist](https://hackmd.io/@freesig/Hk9AmKJNS)

### CODE REVIEW

**Comment by AshantiMutinta** [comment]
This is awesome, does it mean rust sodium has been taken out too? I could get a running conductor in windows easily now hehe

**Comment by neonphog** [comment]
@AshantiMutinta 

> This is awesome, does it mean rust sodium has been taken out too? I could get a running conductor in windows easily now hehe

haha, no, sodium is a core part of our key management/dpki - it's here to stay - it totally builds on windows if we have time to take a breath, we should figure out where the problem is.

File: crates/conductor_lib/src/conductor/base.rs [hunk 1]
```diff
@@ -312,9 +302,7 @@ impl Conductor {
     }
 
     pub fn p2p_bindings(&self) -> Option<Vec<String>> {
-        self.network_spawn
-            .as_ref()
-            .map(|spawn| spawn.p2p_bindings.clone())
+        None
```
Comment (zippy) [inline crates/conductor_lib/src/conductor/base.rs]: I think we need a comment here to explain why this is `None` and how something might be addable back in here in the future?

Comment (neonphog) [reply crates/conductor_lib/src/conductor/base.rs]: actually, turns out this function isn't used anywhere - removed

**Pull request merged**